tool/actions-plan-preview/main.go

```diff
@@ -141,8 +141,33 @@ func main() {
 			doComment(failureBadgeURL + "\nUnable to run plan-preview for a closed pull request.")
 			return
 		}
+
+		// Find comments we sent before
+		comment, err := findLatestPlanPreviewComment(ctx, ghGraphQLClient, event.Owner, event.Repo, event.PRNumber)
+		if err != nil {
+			log.Printf("Unable to find the previous comment to minimize (%v)\n", err)
+		}
+
 		body := makeCommentBody(event, result)
-		doComment(failureBadgeURL + "\n" + body)
+		doComment(body)
+
+		if comment == nil {
+			log.Println("plan-preview result has error")
+			os.Exit(1)
+			return
+		}
```

Comment:
What if there is no previous comment 🤔

Reply:
@khanhtc1202 
I'm sorry for replying later.
Nothing to minimize any message if there is no previous comment.
I'm not sure what you want to know, sorry.

Reply:
I mean in that case, this comment would be nil and I guess we can return normally, no need to exit with error here, isn't it? 🤔

Reply:
@khanhtc1202 
This `comment` is latest previous comment that must close.
But `if result.HasError() {}` is still `true`, so we have to exit with error.

Reply:
@khanhtc1202 cc @TakumaKurosawa 
It should stop as exit 1 because the fix is in the error case in which plan-preview failed.

So if the previous comment doesn't exist, the code wants to finish as exit 1 with any errors when the plan-preview result has some errors.
This is like an early return.

Reply:
This is confusing, so he created a function for minimizing a previous comment.
This function does not affect whether to finish as exit 1.
https://github.com/pipe-cd/pipecd/commit/74a9d8e231a532e2c86ab8e137119dafe86c0988

Reply:
Got your points, thank both 🙏